pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -1192,6 +1209,9 @@ public enum WindowOverFlowMode {
     public static class PlanVersions {
       public static final int V1 = 1;
     }
+
+    public static final String ASK_SERVERS_FOR_EXPLAIN_PLAN = "pinot.query.explain.ask.servers";
+    public static final boolean DEFAULT_ASK_SERVERS_FOR_EXPLAIN_PLAN = true;
```

Comment:
Probably this should be changed to false in the initial version. I'm keeping it as true for a while to verify that all tests are running.